### PR TITLE
[#2328] Display any source in the inbox

### LINKED ISSUE #2328: Display any source in the inbox
## Is your feature request related to a problem? Please describe.

Whenever we add a new [source](https://airy.co/docs/core/getting-started/glossary#source) to Airy the inbox will crash when it receives messages for that source.

This is because there are some places where we hard-code the available sources with a switch ([example](https://github.com/airyhq/airy/blob/b1dd87e30365195345581314566d62c60b87310a/frontend/ui/src/components/IconChannel/index.tsx#L38)).

## Describe the solution you'd like

- [x] Display an unknown source with a default icon
- [x] Render messages for unknown sources by simply displaying the `message.content` of the message as text. If possible we could visually distinguish these messages from messages that can be rendered.
- [x] There's no need to display them on the channels page

### PR DESCRIPTION
**Pull request by AudreyKj**
closes #2328 

**changes:** 
- added unknown source icon for the unknown sources 
- added rendering of messages of unknown sources with small header cue

**screenshots:** 
Screenshot 2021-09-02 at 14 31 39

Screenshot 2021-09-02 at 14 50 21

Screenshot 2021-09-02 at 14 29 25
Screenshot 2021-09-02 at 14 28 21
Screenshot 2021-09-06 at 13 12 22

### CODE REVIEW

File: lib/typescript/render/components/UnknownSourceText/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+type TextRenderProps = {
+  text: string;
+  fromContact?: boolean;
+};
+
+export const UnknownSourceText = ({text, fromContact}: TextRenderProps) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.unknownSourceHeader}>
+        <span className={styles.unknownSource}>Unknown Source</span>
```
Comment (chrismatix) [inline lib/typescript/render/components/UnknownSourceText/index.tsx:14]: Could we here perhaps use the `source` identifier instead? After all, many sources will be "known" just without proper rendering logic :)

**Review by chrismatix — COMMENTED** [review]
Looks super good. Minor comment!

File: lib/typescript/render/components/UnknownSourceText/index.tsx (hunk 1 above)
Comment (chrismatix) [reply lib/typescript/render/components/UnknownSourceText/index.tsx:14]: Or just hardcode: "Raw source content"

File: lib/typescript/render/components/UnknownSourceText/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  sourceName: string;
+};
+
+export const UnknownSourceText = ({text, fromContact, sourceName}: TextRenderProps) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.unknownSourceHeader}>
+        <span className={styles.unknownSource}>{sourceName ?? 'Unknown'} Source</span>
+      </div>
+      <Linkify
```
Comment (AudreyKj) [inline lib/typescript/render/components/UnknownSourceText/index.tsx:17]: here we display the source name of the unknown source

Comment (AudreyKj) [reply lib/typescript/render/components/UnknownSourceText/index.tsx:17]: Screenshot 2021-09-06 at 13 12 22

**Review by chrismatix — APPROVED** [review]
Perfect, thanks!

**Pull request merged**